src/main/java/com/sap/oss/phosphor/fosstars/model/score/oss/OssArtifactSecurityScore.java

```diff
@@ -88,7 +88,9 @@ public ScoreValue calculate(Value<?>... values) {
     }
```

Comment:
I would suggest in case of an unknown score (either artifact or project) to return to combined score as unknown.
Because the `OssArtifactSecurityScore` take the project score and modify it based on the artifact version score.
If the project score is unknown it can not be modified and just returning the artifact version score is not correct (IMHO).

Other opinions?

As sample (which requires changes additional changes in line 91ff):
```suggestion
    } else if (projectSecurityScore.isUnknown()) {
      return scoreValue.withMinConfidence().makeUnknown();
    }
```

Reply:
I agree with your argument. The artifact score should be co-dependent with the ProjectScore.